components/bps-tools/plugins/examples/plugins/org.eclipse.bpel.extensionsample.model/src/org/eclipse/bpel/extensionsample/model/SampleSimpleActivity.java

```diff
@@ -1,54 +0,0 @@
-/**
- * <copyright>
```

Comment:
Don't we need to add WSO2 License in these files? Please check other files as well and make the required changes if needed

Reply:
The file is deleted and no longer in the code base.